src/services/BlobStore.jl

```diff
@@ -134,6 +137,42 @@ end
 
 ## Complete the upload
 
+function completeUpload(
+    client::GQL.Client,
+    blobId::UUID,
+    uploadId::AbstractString,
+    eTags::AbstractVector{<:AbstractString},
+)
+    # FIXME where to get CompletedUploadPartInput CompletedUploadInput
+
+    parts = Vector{CompletedUploadPartInput}()
+    for (pn,eTag) in enumerate(eTags)
+        push!(parts,
+            CompletedUploadPartInput(
+                pn,
+                eTag,
+            )
+        )
+    end
+
+    cui = CompletedUploadInput(
+        uploadId,
+        parts
+    )
+
+    response = GQL.execute(
+        client,
+        GQL_COMPLETEUPLOAD;
+        variables = Dict(
+            "blobId" => blobId, 
+            "completedUpload" => cui, 
+        ),
+        throw_on_execution_error = true,
+    )
```

Comment:
I'm slowly transitioning to the convenience wrapper `executeQql`.

Reply:
did not find easy example so just going with copy from single part upload